src/lib.rs

```diff
@@ -293,6 +225,119 @@ The Mobile Releases Team
 This report was created using `recrep` for {{organization}}/{{application}}/{{version}}."#
     }
 
+    fn no_crashes_found_template<'a>(&self) -> &'a str {
+        r#"
+Hello everyone!
+
+This is the crash newsletter of v{{version}}
+
+Luckily this version does not have any crashes AppCenter knows about. Congratulations 🎉!
+
+This report was created using `recrep` for {{organization}}/{{application}}/{{version}}.
```

Comment:
I like the generic template format.

Reply:
It was the only idea I came up with that separates between the two cases, `no crashes` / `some crash(es)`